__tests__/api.test.js

```diff
@@ -0,0 +1,24 @@
+const request = require('supertest')
+const app = require("../app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end()
+})
+describe("API endpoints", () => {
+    test("/api/topics endpoint should fetch all topics", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+            .expect((result) => {
+            const expected = [{"description": "The man, the Mitch, the legend", "slug": "mitch"}, {"description": "Not dogs", "slug": "cats"}, {"description": "what books are made of", "slug": "paper"}]
+          expect(result.body).toEqual(expected);
```

Comment:
The body of the result should have an appropriate key containing the expected results.

Reply:
I added topics key to the response body.

Reply:
Before pull request, I remember updating this test case, instead of entire response, I said length should be 3 and object at zero index should have slug and description keys. Not sure why earlier version stayed here :)